dictionaries/cpp/dict/cpp.txt

```diff
@@ -6921,7 +6921,7 @@ KC
 KCDATA_TYPE_ARRAY_PADa
 KCDATA_TYPE_ARRAY_PADf
 KCDATA_TYPE_LIBRARY_LOADINFO
-KCDATA_TYPE_TYPEDEFINTION
+KCDATA_TYPE_TYPEDEFINITION
```

Comment:
Guessing this was extracted with a typo. Not sure if it would be better to delete entirely

Reply:
Will removing it break some CI in unknown repos?

Reply:
I doubt it, so it's probably fine to leave the corrected one too